### PR TITLE
Bump prettier to 3.3.3

### PR DESCRIPTION
**Pull request by aytey**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -10410,20 +10409,6 @@
         "url": "https://github.com/sponsors/sindresorhus"
       }
     },
-    "node_modules/xo/node_modules/prettier": {
```
Comment (aytey) [inline package-lock.json:10413]: I don't know where this has gone or if it is a problem.

Comment (Zbigor) [reply package-lock.json:10413]: I guess xo "installs it" and "gets an override. If we agree to use `npx prettier -w` for now, it should be fine. Whenever we actually get to using `xo`, we'll define everything there, including `eslint` config etc.

Comment (aytey) [reply package-lock.json:10413]: I've asked @SalvatoreZagaria to add a new check that does `npx prettier --check $(echo **/*.ts | tr ' ' '\n' | grep -v "node_modules" | xargs)`.

Comment (SalvatoreZagaria) [reply package-lock.json:10413]: https://github.com/vectorgrp/vector-vscode-vcast/pull/161

**Pull request merged**